.github/workflows/ci-rs.yml

```diff
@@ -91,9 +93,9 @@ jobs:
       - name: Install CapnProto
         run: sudo apt-get install -y capnproto
       - name: Build benchmarks with no features
-        run: cargo bench --verbose --no-run --workspace --no-default-features
+        run: cargo bench --verbose --no-run --workspace --exclude hugr-llvm --no-default-features
       - name: Build benchmarks with all features
-        run: cargo bench --verbose --no-run --workspace --all-features
+        run: cargo bench --verbose --no-run --workspace --exclude hugr-llvm --all-features
```

Comment:
Won't `--all-features` include `hugr-llvm` anyways, since it's an optional dep of `hugr`?

Reply:
yes you're right, the excluding only makes sense for tests